cmd/tikv-ctl/src/main.rs

```diff
@@ -831,43 +835,68 @@ fn flashback_whole_cluster(
         .unwrap();
 
     block_on(runtime.spawn(async move {
-        // Prepare all regions for flashback.
+        // Get all stores related debug executor.
+        let stores = match pd_client.get_all_stores(false) {
+            Ok(stores) => stores,
+            Err(e) => {
+                println!("failed to load all stores: {:?}", e);
+                return;
+            }
+        };
+        let debuggers = Mutex::new(stores
+            .into_iter()
+            .map(|s| {
+                let addr = pd_client.get_store(s.get_id()).unwrap().address;
+                let cfg_inner = cfg.clone();
+                let mgr = Arc::clone(&mgr);
+                let debug_executor = new_debug_executor(&cfg_inner, None, Some(&addr), mgr);
+                (s.get_id(), debug_executor)
+            } )
+            .collect::<HashMap<_, _>>());
+        // Prepare flashback.
         let start_ts = pd_client.get_tso().await.unwrap();
```

Comment:
Better add a TODO here to indicate we could manually pass the `start_ts` and `commit_ts` later.

Reply:
If specified `ts` manually there seems to be no guarantee that it will always be the maximum and that no other written, which will destroy https://github.com/tikv/tikv/blob/master/src/storage/txn/actions/flashback_to_version.rs#L45 assert.